docs/auditors/mounts.md

```diff
@@ -0,0 +1,273 @@
+| /var/run/docker.sock      |  Unix socket used to communicate with Docker daemon                                   |
+| /                         |  Filesystem's root |
+| /etc                      |  Directory that usually contains all system related configurations files         |
+| /root                     |  Home directory of the `root` user                                             |
+| /var/run/crio/crio.sock   |  Unix socket used to communicate with the CRI-O Container Engine                                                 |
+| /home/admin               |  Home directory of the `admin` user        |
+| /var/lib/kubelet          |  Directory for Kublet-related configuration                                                             |
+| /var/lib/kubelet/pki      |  Directory containing the certificate and private key of the Kublet                                                               |
+| /etc/kubernetes           |  Directory containing Kubernetes related configuration              |
+| /etc/kubernetes/manifests |  Directory containing manifest of Kubernetes components                                                         |
+
+## Examples
+
+```
+$ kubeaudit mounts -f auditors/mounts/fixtures/proc-mounted.yml
+
+---------------- Results for ---------------
+
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: pod
+    namespace: proc-mounted
+
+--------------------------------------------
+
+-- [error] SensitivePathsMounted
+   Message: Sensitive path mounted as volume: proc-volume (/proc -> /host/proc, readOnly: false). It should be removed from the container's mounts list.
+   Metadata:
+      Container: container
+      MountName: proc-volume
+      MountPath: /host/proc
+      MountReadOnly: false
+      MountVolume: proc-volume
+      MountVolumeHostPath: /proc
+
+```
+
+### Example with Config File
+
+A custom paths list can be provided in the config file. See [docs](docs/all.md) for more information. These are the host
+paths you'd like to have kubeaudit raise an error when they are mounted in a container.
+
+`config.yaml`
+
+```yaml
+---
+auditors:
+  mounts:
+    paths: ["/etc", "/var/run/docker.sock"]
+```
+
+`manifest.yaml`
+
+```yaml
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: deployment
+  namespace: example-namespace
+spec:
+  template:
+    spec:
+      containers:
+        - name: container
+          image: scratch
+          volumeMounts:
+            - mountPath: /host/etc
+              name: etc-volume
+            - mountPath: /var/run/docker.sock
+              name: docker-socket-volume
+      volumes:
+        - name: etc-volume
+          hostPath:
+            path: /etc
+        - name: docker-socket-volume
+          hostPath:
+            path: /var/run/docker.sock
+```
+
+```shell
+$ kubeaudit mounts --kconfig "config.yaml" -f "manifest.yaml"
```

Comment:
Passing in a config file only works with `all` and `autofix`(https://github.com/Shopify/kubeaudit#configuration-file)

```suggestion
$ kubeaudit all --kconfig  "config.yaml" -f "manifest.yaml"
```